src/parser.ts

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-import { substrait } from "../dist/definitions"
+import { substrait } from "./generated/definitions"
```

Comment:
Importing from ../dist won't work at runtime.  That's why I needed to move things to src.